lib/Docs/RSTBuilder.php

```diff
@@ -141,7 +141,7 @@ private function copyRst(Project $project, ProjectVersion $version) : void
         // check if we have an explicit sidebar file to use
         // otherwise just use the default autogenerated sidebar
         $sidebarPath = $docsPath . '/sidebar.rst';
-        $sidebar     = file_exists($sidebarPath) ? file_get_contents($sidebarPath) : self::DEFAULT_SIDEBAR;
+        $sidebar     = file_exists($sidebarPath) ? (string) file_get_contents($sidebarPath) : self::DEFAULT_SIDEBAR;
```

Comment:
Same here, assertions. Maybe make your own `file_get_contents()` that throws and always returns `string`?